pkg/didcomm/messenger/messenger.go

```diff
@@ -7,46 +7,155 @@ SPDX-License-Identifier: Apache-2.0
 package messenger
 
 import (
+	"encoding/json"
 	"errors"
+	"fmt"
 
+	"github.com/google/uuid"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+const (
+	messengerStore = "messenger_store"
+
+	jsonID             = "@id"
+	jsonThread         = "~thread"
```

Comment:
can't we export these constants from `pkg/didcomm/common/service/did_comm_msg.go`?

Reply:
I'll keep it in mind. I don't want to export it. It will look like a mess. In the messenger, I am gonna use a lot of JSON tags. And import two tags from another package looks awful to me. Better to keep it together. For now, it will be separated. But, I will come up with how to merge them together. It might be related to moving the current DidCommMsg implementation under the messenger folder.